CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Feature Requests
+Want a new feature in Mudlet? Let us know about it! We also have a long-running feature requests thread on [our forums](http://forums.mudlet.org/viewtopic.php?f=5&t=92).
+
+# Bug Reports
```

Comment:
Do we actually need to say something about how to raise a bug report - i.e. raise an issue here about it?